background/providers/kat.js

```diff
@@ -0,0 +1,103 @@
+var https = require('https');
+var zlib = require('zlib');
+var path = require('path');
+var nconf = require('nconf');
+var mongoose = require('mongoose');
+var Category = require('../../app/models/Category.js');
+var Torrent = require('../../app/models/Torrent.js');
```

Comment:
We should look into using the same thing I did for the app.js file. This allows us to have /app as a relative root.
````
require('app-module-path').addPath(__dirname + '/app');
````